sdk/javascript/index.js

```diff
@@ -20,5 +20,6 @@
 module.exports = {
   processor: require('./processor'),
   exceptions: require('./processor/exceptions'),
```

Comment:
Should the signing errors be included here?

If not maybe this should be removed and there should be a `signing.errors` and a `processor.errors`

Reply:
The are exported again into  `./signing/index.js`, since usage, in browser, would probably be doing:

```
let signing = require('sawtooth-sdk/signing')
...
```
to avoid the issues with ZMQ